modules/hostap/src/supp_main.c

```diff
@@ -94,7 +94,6 @@ DEFINE_WIFI_NM_INSTANCE(wifi_supplicant, &mgmt_ops);
 
 #ifdef CONFIG_WIFI_NM_HOSTAPD_AP
 static const struct wifi_mgmt_ops mgmt_ap_ops = {
-	.set_btwt = supplicant_set_btwt,
```

Comment:
same as AP bandwidth, an accidental change now being reverted?

Reply:
this is caused by wrong upstream sequence from downstream